web/src/js/ducks/settings.js

```diff
@@ -1,64 +1,72 @@
-import { addLogEntry } from './eventLog'
+import {fetchApi} from "../utils";
 
-export const WS_MSG_TYPE = 'settings'
-export const WS_MSG_CMD_UPDATE = 'update'
```

Comment:
`updateSettings` is the only occasion where we'd ever want to use them, so I think this is fine. If we ever come up with a second usage, we should put it in a constant ASAP.